### ISSUE TITLE
CI failure: Marshalling of type object is not implemented - Constructor parameters did not have matching fixture data: OleTxFixture fixture

### ISSUE THREAD

**Issue by carlossanlop** [dotnet/runtime#74187]
A bunch of System.Transactions tests failed in with this message in the callstack:

```
      System.AggregateException : One or more errors occurred. (Marshalling of type object is not implemented) (The following constructor parameters did not have matching fixture data: OleTxFixture fixture)
      ---- System.Runtime.InteropServices.MarshalDirectiveException : Marshalling of type object is not imp
```

- This was found in an RC1 PR: https://github.com/dotnet/runtime/pull/74045
- Pipeline: `runtime-extra-platforms`
- Job: [Build windows x64 Release AllSubsets_Mono](https://dev.azure.com/dnceng/public/_build/results?buildId=1951942&view=logs&j=585f1246-0618-5d47-ff3e-08c83309e2d2&t=5e10daf8-cc46-5bd4-2ab7-d28eafd857b5&l=89)
- [Results of all failed tests](https://helixre107v0xdeko0k025g8.blob.core.windows.net/dotnet-runtime-refs-heads-release-70-rc1-75706d1e4eff4cbea0/System.Transactions.Local.Tests/1/console.8683904c.log?helixlogtype=result)):
```
  System.Transactions.Tests.OleTxTests.Volatile_and_durable_enlistments(volatileCount: 0) [FAIL]
      System.AggregateException : One or more errors occurred. (Marshalling of type object is not implemented) (The following constructor parameters did not have matching fixture data: OleTxFixture fixture)
      ---- System.Runtime.InteropServices.MarshalDirectiveException : Marshalling of type object is not implemented
      ---- The following constructor parameters did not have matching fixture data: OleTxFixture fixture
      Stack Trace:
        
        ----- Inner Stack Trace #1 (System.Runtime.InteropServices.MarshalDirectiveException) -----
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/DtcProxyShim/DtcProxyShimFactory.cs(66,0): at System.Transactions.DtcProxyShim.DtcProxyShimFactory.ConnectToProxy(String nodeName, Guid resourceManagerIdentifier, Object managedIdentifier, Boolean& nodeNameMatches, Byte[]& whereabouts, ResourceManagerShim& resourceManagerShim)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Oletx/DtcTransactionManager.cs(38,0): at System.Transactions.Oletx.DtcTransactionManager.Initialize()
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Oletx/DtcTransactionManager.cs(79,0): at System.Transactions.Oletx.DtcTransactionManager.get_ProxyShimFactory()
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Oletx/OletxTransactionManager.cs(448,0): at System.Transactions.Oletx.OletxTransactionManager.CreateTransaction(TransactionOptions properties)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/TransactionState.cs(2172,0): at System.Transactions.TransactionStatePromoted.EnterState(InternalTransaction tx)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/TransactionState.cs(549,0): at System.Transactions.EnlistableStates.EnlistDurable(InternalTransaction tx, Guid resourceManagerIdentifier, IEnlistmentNotification enlistmentNotification, EnlistmentOptions enlistmentOptions, Transaction atomicTransaction)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Transaction.cs(503,0): at System.Transactions.Transaction.EnlistDurable(Guid resourceManagerIdentifier, IEnlistmentNotification enlistmentNotification, EnlistmentOptions enlistmentOptions)
        /_/src/libraries/System.Transactions.Local/tests/OleTxTests.cs(504,0): at System.Transactions.Tests.OleTxTests.OleTxFixture..ctor()
        /_/src/mono/System.Private.CoreLib/src/System/Reflection/ConstructorInvoker.Mono.cs(20,0): at System.Reflection.ConstructorInvoker.InterpretedInvoke(Object obj, Span`1 args, BindingFlags invokeAttr)
        ----- Inner Stack Trace #2 (Xunit.Sdk.TestClassException) -----
        
    System.Transactions.Tests.OleTxTests.Volatile_and_durable_enlistments(volatileCount: 1) [FAIL]
      System.AggregateException : One or more errors occurred. (Marshalling of type object is not implemented) (The following constructor parameters did not have matching fixture data: OleTxFixture fixture)
      ---- System.Runtime.InteropServices.MarshalDirectiveException : Marshalling of type object is not implemented
      ---- The following constructor parameters did not have matching fixture data: OleTxFixture fixture
      Stack Trace:
        
        ----- Inner Stack Trace #1 (System.Runtime.InteropServices.MarshalDirectiveException) -----
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/DtcProxyShim/DtcProxyShimFactory.cs(66,0): at System.Transactions.DtcProxyShim.DtcProxyShimFactory.ConnectToProxy(String nodeName, Guid resourceManagerIdentifier, Object managedIdentifier, Boolean& nodeNameMatches, Byte[]& whereabouts, ResourceManagerShim& resourceManagerShim)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Oletx/DtcTransactionManager.cs(38,0): at System.Transactions.Oletx.DtcTransactionManager.Initialize()
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Oletx/DtcTransactionManager.cs(79,0): at System.Transactions.Oletx.DtcTransactionManager.get_ProxyShimFactory()
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Oletx/OletxTransactionManager.cs(448,0): at System.Transactions.Oletx.OletxTransactionManager.CreateTransaction(TransactionOptions properties)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/TransactionState.cs(2172,0): at System.Transactions.TransactionStatePromoted.EnterState(InternalTransaction tx)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/TransactionState.cs(549,0): at System.Transactions.EnlistableStates.EnlistDurable(InternalTransaction tx, Guid resourceManagerIdentifier, IEnlistmentNotification enlistmentNotification, EnlistmentOptions enlistmentOptions, Transaction atomicTransaction)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Transaction.cs(503,0): at System.Transactions.Transaction.EnlistDurable(Guid resourceManagerIdentifier, IEnlistmentNotification enlistmentNotification, EnlistmentOptions enlistmentOptions)
        /_/src/libraries/System.Transactions.Local/tests/OleTxTests.cs(504,0): at System.Transactions.Tests.OleTxTests.OleTxFixture..ctor()
        /_/src/mono/System.Private.CoreLib/src/System/Reflection/ConstructorInvoker.Mono.cs(20,0): at System.Reflection.ConstructorInvoker.InterpretedInvoke(Object obj, Span`1 args, BindingFlags invokeAttr)
        ----- Inner Stack Trace #2 (Xunit.Sdk.TestClassException) -----
        
    System.Transactions.Tests.OleTxTests.Volatile_and_durable_enlistments(volatileCount: 2) [FAIL]
      System.AggregateException : One or more errors occurred. (Marshalling of type object is not implemented) (The following constructor parameters did not have matching fixture data: OleTxFixture fixture)
      ---- System.Runtime.InteropServices.MarshalDirectiveException : Marshalling of type object is not implemented
      ---- The following constructor parameters did not have matching fixture data: OleTxFixture fixture
      Stack Trace:
        
        ----- Inner Stack Trace #1 (System.Runtime.InteropServices.MarshalDirectiveException) -----
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/DtcProxyShim/DtcProxyShimFactory.cs(66,0): at System.Transactions.DtcProxyShim.DtcProxyShimFactory.ConnectToProxy(String nodeName, Guid resourceManagerIdentifier, Object managedIdentifier, Boolean& nodeNameMatches, Byte[]& whereabouts, ResourceManagerShim& resourceManagerShim)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Oletx/DtcTransactionManager.cs(38,0): at System.Transactions.Oletx.DtcTransactionManager.Initialize()
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Oletx/DtcTransactionManager.cs(79,0): at System.Transactions.Oletx.DtcTransactionManager.get_ProxyShimFactory()
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Oletx/OletxTransactionManager.cs(448,0): at System.Transactions.Oletx.OletxTransactionManager.CreateTransaction(TransactionOptions properties)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/TransactionState.cs(2172,0): at System.Transactions.TransactionStatePromoted.EnterState(InternalTransaction tx)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/TransactionState.cs(549,0): at System.Transactions.EnlistableStates.EnlistDurable(InternalTransaction tx, Guid resourceManagerIdentifier, IEnlistmentNotification enlistmentNotification, EnlistmentOptions enlistmentOptions, Transaction atomicTransaction)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Transaction.cs(503,0): at System.Transactions.Transaction.EnlistDurable(Guid resourceManagerIdentifier, IEnlistmentNotification enlistmentNotification, EnlistmentOptions enlistmentOptions)
        /_/src/libraries/System.Transactions.Local/tests/OleTxTests.cs(504,0): at System.Transactions.Tests.OleTxTests.OleTxFixture..ctor()
        /_/src/mono/System.Private.CoreLib/src/System/Reflection/ConstructorInvoker.Mono.cs(20,0): at System.Reflection.ConstructorInvoker.InterpretedInvoke(Object obj, Span`1 args, BindingFlags invokeAttr)
        ----- Inner Stack Trace #2 (Xunit.Sdk.TestClassException) -----
        
    System.Transactions.Tests.OleTxTests.TransmitterPropagationToken [FAIL]
      System.AggregateException : One or more errors occurred. (Marshalling of type object is not implemented) (The following constructor parameters did not have matching fixture data: OleTxFixture fixture)
      ---- System.Runtime.InteropServices.MarshalDirectiveException : Marshalling of type object is not implemented
      ---- The following constructor parameters did not have matching fixture data: OleTxFixture fixture
      Stack Trace:
        
        ----- Inner Stack Trace #1 (System.Runtime.InteropServices.MarshalDirectiveException) -----
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/DtcProxyShim/DtcProxyShimFactory.cs(66,0): at System.Transactions.DtcProxyShim.DtcProxyShimFactory.ConnectToProxy(String nodeName, Guid resourceManagerIdentifier, Object managedIdentifier, Boolean& nodeNameMatches, Byte[]& whereabouts, ResourceManagerShim& resourceManagerShim)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Oletx/DtcTransactionManager.cs(38,0): at System.Transactions.Oletx.DtcTransactionManager.Initialize()
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Oletx/DtcTransactionManager.cs(79,0): at System.Transactions.Oletx.DtcTransactionManager.get_ProxyShimFactory()
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Oletx/OletxTransactionManager.cs(448,0): at System.Transactions.Oletx.OletxTransactionManager.CreateTransaction(TransactionOptions properties)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/TransactionState.cs(2172,0): at System.Transactions.TransactionStatePromoted.EnterState(InternalTransaction tx)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/TransactionState.cs(549,0): at System.Transactions.EnlistableStates.EnlistDurable(InternalTransaction tx, Guid resourceManagerIdentifier, IEnlistmentNotification enlistmentNotification, EnlistmentOptions enlistmentOptions, Transaction atomicTransaction)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Transaction.cs(503,0): at System.Transactions.Transaction.EnlistDurable(Guid resourceManagerIdentifier, IEnlistmentNotification enlistmentNotification, EnlistmentOptions enlistmentOptions)
        /_/src/libraries/System.Transactions.Local/tests/OleTxTests.cs(504,0): at System.Transactions.Tests.OleTxTests.OleTxFixture..ctor()
        /_/src/mono/System.Private.CoreLib/src/System/Reflection/ConstructorInvoker.Mono.cs(20,0): at System.Reflection.ConstructorInvoker.InterpretedInvoke(Object obj, Span`1 args, BindingFlags invokeAttr)
        ----- Inner Stack Trace #2 (Xunit.Sdk.TestClassException) -----
        
    System.Transactions.Tests.OleTxTests.GetExportCookie [FAIL]
      System.AggregateException : One or more errors occurred. (Marshalling of type object is not implemented) (The following constructor parameters did not have matching fixture data: OleTxFixture fixture)
      ---- System.Runtime.InteropServices.MarshalDirectiveException : Marshalling of type object is not implemented
      ---- The following constructor parameters did not have matching fixture data: OleTxFixture fixture
      Stack Trace:
        
        ----- Inner Stack Trace #1 (System.Runtime.InteropServices.MarshalDirectiveException) -----
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/DtcProxyShim/DtcProxyShimFactory.cs(66,0): at System.Transactions.DtcProxyShim.DtcProxyShimFactory.ConnectToProxy(String nodeName, Guid resourceManagerIdentifier, Object managedIdentifier, Boolean& nodeNameMatches, Byte[]& whereabouts, ResourceManagerShim& resourceManagerShim)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Oletx/DtcTransactionManager.cs(38,0): at System.Transactions.Oletx.DtcTransactionManager.Initialize()
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Oletx/DtcTransactionManager.cs(79,0): at System.Transactions.Oletx.DtcTransactionManager.get_ProxyShimFactory()
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Oletx/OletxTransactionManager.cs(448,0): at System.Transactions.Oletx.OletxTransactionManager.CreateTransaction(TransactionOptions properties)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/TransactionState.cs(2172,0): at System.Transactions.TransactionStatePromoted.EnterState(InternalTransaction tx)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/TransactionState.cs(549,0): at System.Transactions.EnlistableStates.EnlistDurable(InternalTransaction tx, Guid resourceManagerIdentifier, IEnlistmentNotification enlistmentNotification, EnlistmentOptions enlistmentOptions, Transaction atomicTransaction)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Transaction.cs(503,0): at System.Transactions.Transaction.EnlistDurable(Guid resourceManagerIdentifier, IEnlistmentNotification enlistmentNotification, EnlistmentOptions enlistmentOptions)
        /_/src/libraries/System.Transactions.Local/tests/OleTxTests.cs(504,0): at System.Transactions.Tests.OleTxTests.OleTxFixture..ctor()
        /_/src/mono/System.Private.CoreLib/src/System/Reflection/ConstructorInvoker.Mono.cs(20,0): at System.Reflection.ConstructorInvoker.InterpretedInvoke(Object obj, Span`1 args, BindingFlags invokeAttr)
        ----- Inner Stack Trace #2 (Xunit.Sdk.TestClassException) -----
        
    System.Transactions.Tests.OleTxTests.Recovery [FAIL]
      System.AggregateException : One or more errors occurred. (Marshalling of type object is not implemented) (The following constructor parameters did not have matching fixture data: OleTxFixture fixture)
      ---- System.Runtime.InteropServices.MarshalDirectiveException : Marshalling of type object is not implemented
      ---- The following constructor parameters did not have matching fixture data: OleTxFixture fixture
      Stack Trace:
        
        ----- Inner Stack Trace #1 (System.Runtime.InteropServices.MarshalDirectiveException) -----
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/DtcProxyShim/DtcProxyShimFactory.cs(66,0): at System.Transactions.DtcProxyShim.DtcProxyShimFactory.ConnectToProxy(String nodeName, Guid resourceManagerIdentifier, Object managedIdentifier, Boolean& nodeNameMatches, Byte[]& whereabouts, ResourceManagerShim& resourceManagerShim)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Oletx/DtcTransactionManager.cs(38,0): at System.Transactions.Oletx.DtcTransactionManager.Initialize()
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Oletx/DtcTransactionManager.cs(79,0): at System.Transactions.Oletx.DtcTransactionManager.get_ProxyShimFactory()
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Oletx/OletxTransactionManager.cs(448,0): at System.Transactions.Oletx.OletxTransactionManager.CreateTransaction(TransactionOptions properties)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/TransactionState.cs(2172,0): at System.Transactions.TransactionStatePromoted.EnterState(InternalTransaction tx)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/TransactionState.cs(549,0): at System.Transactions.EnlistableStates.EnlistDurable(InternalTransaction tx, Guid resourceManagerIdentifier, IEnlistmentNotification enlistmentNotification, EnlistmentOptions enlistmentOptions, Transaction atomicTransaction)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Transaction.cs(503,0): at System.Transactions.Transaction.EnlistDurable(Guid resourceManagerIdentifier, IEnlistmentNotification enlistmentNotification, EnlistmentOptions enlistmentOptions)
        /_/src/libraries/System.Transactions.Local/tests/OleTxTests.cs(504,0): at System.Transactions.Tests.OleTxTests.OleTxFixture..ctor()
        /_/src/mono/System.Private.CoreLib/src/System/Reflection/ConstructorInvoker.Mono.cs(20,0): at System.Reflection.ConstructorInvoker.InterpretedInvoke(Object obj, Span`1 args, BindingFlags invokeAttr)
        ----- Inner Stack Trace #2 (Xunit.Sdk.TestClassException) -----
        
    System.Transactions.Tests.OleTxTests.Two_durable_enlistments_rollback [FAIL]
      System.AggregateException : One or more errors occurred. (Marshalling of type object is not implemented) (The following constructor parameters did not have matching fixture data: OleTxFixture fixture)
      ---- System.Runtime.InteropServices.MarshalDirectiveException : Marshalling of type object is not implemented
      ---- The following constructor parameters did not have matching fixture data: OleTxFixture fixture
      Stack Trace:
        
        ----- Inner Stack Trace #1 (System.Runtime.InteropServices.MarshalDirectiveException) -----
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/DtcProxyShim/DtcProxyShimFactory.cs(66,0): at System.Transactions.DtcProxyShim.DtcProxyShimFactory.ConnectToProxy(String nodeName, Guid resourceManagerIdentifier, Object managedIdentifier, Boolean& nodeNameMatches, Byte[]& whereabouts, ResourceManagerShim& resourceManagerShim)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Oletx/DtcTransactionManager.cs(38,0): at System.Transactions.Oletx.DtcTransactionManager.Initialize()
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Oletx/DtcTransactionManager.cs(79,0): at System.Transactions.Oletx.DtcTransactionManager.get_ProxyShimFactory()
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Oletx/OletxTransactionManager.cs(448,0): at System.Transactions.Oletx.OletxTransactionManager.CreateTransaction(TransactionOptions properties)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/TransactionState.cs(2172,0): at System.Transactions.TransactionStatePromoted.EnterState(InternalTransaction tx)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/TransactionState.cs(549,0): at System.Transactions.EnlistableStates.EnlistDurable(InternalTransaction tx, Guid resourceManagerIdentifier, IEnlistmentNotification enlistmentNotification, EnlistmentOptions enlistmentOptions, Transaction atomicTransaction)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Transaction.cs(503,0): at System.Transactions.Transaction.EnlistDurable(Guid resourceManagerIdentifier, IEnlistmentNotification enlistmentNotification, EnlistmentOptions enlistmentOptions)
        /_/src/libraries/System.Transactions.Local/tests/OleTxTests.cs(504,0): at System.Transactions.Tests.OleTxTests.OleTxFixture..ctor()
        /_/src/mono/System.Private.CoreLib/src/System/Reflection/ConstructorInvoker.Mono.cs(20,0): at System.Reflection.ConstructorInvoker.InterpretedInvoke(Object obj, Span`1 args, BindingFlags invokeAttr)
        ----- Inner Stack Trace #2 (Xunit.Sdk.TestClassException) -----
        
    System.Transactions.Tests.OleTxTests.Two_durable_enlistments_commit(vote1: Prepared, vote2: Prepared, expectedOutcome1: Committed, expectedOutcome2: Committed, expectedTxStatus: Committed) [FAIL]
      System.AggregateException : One or more errors occurred. (Marshalling of type object is not implemented) (The following constructor parameters did not have matching fixture data: OleTxFixture fixture)
      ---- System.Runtime.InteropServices.MarshalDirectiveException : Marshalling of type object is not implemented
      ---- The following constructor parameters did not have matching fixture data: OleTxFixture fixture
      Stack Trace:
        
        ----- Inner Stack Trace #1 (System.Runtime.InteropServices.MarshalDirectiveException) -----
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/DtcProxyShim/DtcProxyShimFactory.cs(66,0): at System.Transactions.DtcProxyShim.DtcProxyShimFactory.ConnectToProxy(String nodeName, Guid resourceManagerIdentifier, Object managedIdentifier, Boolean& nodeNameMatches, Byte[]& whereabouts, ResourceManagerShim& resourceManagerShim)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Oletx/DtcTransactionManager.cs(38,0): at System.Transactions.Oletx.DtcTransactionManager.Initialize()
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Oletx/DtcTransactionManager.cs(79,0): at System.Transactions.Oletx.DtcTransactionManager.get_ProxyShimFactory()
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Oletx/OletxTransactionManager.cs(448,0): at System.Transactions.Oletx.OletxTransactionManager.CreateTransaction(TransactionOptions properties)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/TransactionState.cs(2172,0): at System.Transactions.TransactionStatePromoted.EnterState(InternalTransaction tx)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/TransactionState.cs(549,0): at System.Transactions.EnlistableStates.EnlistDurable(InternalTransaction tx, Guid resourceManagerIdentifier, IEnlistmentNotification enlistmentNotification, EnlistmentOptions enlistmentOptions, Transaction atomicTransaction)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Transaction.cs(503,0): at System.Transactions.Transaction.EnlistDurable(Guid resourceManagerIdentifier, IEnlistmentNotification enlistmentNotification, EnlistmentOptions enlistmentOptions)
        /_/src/libraries/System.Transactions.Local/tests/OleTxTests.cs(504,0): at System.Transactions.Tests.OleTxTests.OleTxFixture..ctor()
        /_/src/mono/System.Private.CoreLib/src/System/Reflection/ConstructorInvoker.Mono.cs(20,0): at System.Reflection.ConstructorInvoker.InterpretedInvoke(Object obj, Span`1 args, BindingFlags invokeAttr)
        ----- Inner Stack Trace #2 (Xunit.Sdk.TestClassException) -----
        
    System.Transactions.Tests.OleTxTests.Two_durable_enlistments_commit(vote1: Prepared, vote2: ForceRollback, expectedOutcome1: Aborted, expectedOutcome2: Aborted, expectedTxStatus: Aborted) [FAIL]
      System.AggregateException : One or more errors occurred. (Marshalling of type object is not implemented) (The following constructor parameters did not have matching fixture data: OleTxFixture fixture)
      ---- System.Runtime.InteropServices.MarshalDirectiveException : Marshalling of type object is not implemented
      ---- The following constructor parameters did not have matching fixture data: OleTxFixture fixture
      Stack Trace:
        
        ----- Inner Stack Trace #1 (System.Runtime.InteropServices.MarshalDirectiveException) -----
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/DtcProxyShim/DtcProxyShimFactory.cs(66,0): at System.Transactions.DtcProxyShim.DtcProxyShimFactory.ConnectToProxy(String nodeName, Guid resourceManagerIdentifier, Object managedIdentifier, Boolean& nodeNameMatches, Byte[]& whereabouts, ResourceManagerShim& resourceManagerShim)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Oletx/DtcTransactionManager.cs(38,0): at System.Transactions.Oletx.DtcTransactionManager.Initialize()
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Oletx/DtcTransactionManager.cs(79,0): at System.Transactions.Oletx.DtcTransactionManager.get_ProxyShimFactory()
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Oletx/OletxTransactionManager.cs(448,0): at System.Transactions.Oletx.OletxTransactionManager.CreateTransaction(TransactionOptions properties)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/TransactionState.cs(2172,0): at System.Transactions.TransactionStatePromoted.EnterState(InternalTransaction tx)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/TransactionState.cs(549,0): at System.Transactions.EnlistableStates.EnlistDurable(InternalTransaction tx, Guid resourceManagerIdentifier, IEnlistmentNotification enlistmentNotification, EnlistmentOptions enlistmentOptions, Transaction atomicTransaction)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Transaction.cs(503,0): at System.Transactions.Transaction.EnlistDurable(Guid resourceManagerIdentifier, IEnlistmentNotification enlistmentNotification, EnlistmentOptions enlistmentOptions)
        /_/src/libraries/System.Transactions.Local/tests/OleTxTests.cs(504,0): at System.Transactions.Tests.OleTxTests.OleTxFixture..ctor()
        /_/src/mono/System.Private.CoreLib/src/System/Reflection/ConstructorInvoker.Mono.cs(20,0): at System.Reflection.ConstructorInvoker.InterpretedInvoke(Object obj, Span`1 args, BindingFlags invokeAttr)
        ----- Inner Stack Trace #2 (Xunit.Sdk.TestClassException) -----
        
    System.Transactions.Tests.OleTxTests.Two_durable_enlistments_commit(vote1: ForceRollback, vote2: Prepared, expectedOutcome1: Aborted, expectedOutcome2: Aborted, expectedTxStatus: Aborted) [FAIL]
      System.AggregateException : One or more errors occurred. (Marshalling of type object is not implemented) (The following constructor parameters did not have matching fixture data: OleTxFixture fixture)
      ---- System.Runtime.InteropServices.MarshalDirectiveException : Marshalling of type object is not implemented
      ---- The following constructor parameters did not have matching fixture data: OleTxFixture fixture
      Stack Trace:
        
        ----- Inner Stack Trace #1 (System.Runtime.InteropServices.MarshalDirectiveException) -----
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/DtcProxyShim/DtcProxyShimFactory.cs(66,0): at System.Transactions.DtcProxyShim.DtcProxyShimFactory.ConnectToProxy(String nodeName, Guid resourceManagerIdentifier, Object managedIdentifier, Boolean& nodeNameMatches, Byte[]& whereabouts, ResourceManagerShim& resourceManagerShim)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Oletx/DtcTransactionManager.cs(38,0): at System.Transactions.Oletx.DtcTransactionManager.Initialize()
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Oletx/DtcTransactionManager.cs(79,0): at System.Transactions.Oletx.DtcTransactionManager.get_ProxyShimFactory()
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Oletx/OletxTransactionManager.cs(448,0): at System.Transactions.Oletx.OletxTransactionManager.CreateTransaction(TransactionOptions properties)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/TransactionState.cs(2172,0): at System.Transactions.TransactionStatePromoted.EnterState(InternalTransaction tx)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/TransactionState.cs(549,0): at System.Transactions.EnlistableStates.EnlistDurable(InternalTransaction tx, Guid resourceManagerIdentifier, IEnlistmentNotification enlistmentNotification, EnlistmentOptions enlistmentOptions, Transaction atomicTransaction)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Transaction.cs(503,0): at System.Transactions.Transaction.EnlistDurable(Guid resourceManagerIdentifier, IEnlistmentNotification enlistmentNotification, EnlistmentOptions enlistmentOptions)
        /_/src/libraries/System.Transactions.Local/tests/OleTxTests.cs(504,0): at System.Transactions.Tests.OleTxTests.OleTxFixture..ctor()
        /_/src/mono/System.Private.CoreLib/src/System/Reflection/ConstructorInvoker.Mono.cs(20,0): at System.Reflection.ConstructorInvoker.InterpretedInvoke(Object obj, Span`1 args, BindingFlags invokeAttr)
        ----- Inner Stack Trace #2 (Xunit.Sdk.TestClassException) -----
        
    System.Transactions.Tests.OleTxTests.Promotion [FAIL]
      System.AggregateException : One or more errors occurred. (Marshalling of type object is not implemented) (The following constructor parameters did not have matching fixture data: OleTxFixture fixture)
      ---- System.Runtime.InteropServices.MarshalDirectiveException : Marshalling of type object is not implemented
      ---- The following constructor parameters did not have matching fixture data: OleTxFixture fixture
      Stack Trace:
        
        ----- Inner Stack Trace #1 (System.Runtime.InteropServices.MarshalDirectiveException) -----
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/DtcProxyShim/DtcProxyShimFactory.cs(66,0): at System.Transactions.DtcProxyShim.DtcProxyShimFactory.ConnectToProxy(String nodeName, Guid resourceManagerIdentifier, Object managedIdentifier, Boolean& nodeNameMatches, Byte[]& whereabouts, ResourceManagerShim& resourceManagerShim)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Oletx/DtcTransactionManager.cs(38,0): at System.Transactions.Oletx.DtcTransactionManager.Initialize()
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Oletx/DtcTransactionManager.cs(79,0): at System.Transactions.Oletx.DtcTransactionManager.get_ProxyShimFactory()
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Oletx/OletxTransactionManager.cs(448,0): at System.Transactions.Oletx.OletxTransactionManager.CreateTransaction(TransactionOptions properties)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/TransactionState.cs(2172,0): at System.Transactions.TransactionStatePromoted.EnterState(InternalTransaction tx)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/TransactionState.cs(549,0): at System.Transactions.EnlistableStates.EnlistDurable(InternalTransaction tx, Guid resourceManagerIdentifier, IEnlistmentNotification enlistmentNotification, EnlistmentOptions enlistmentOptions, Transaction atomicTransaction)
        /_/src/libraries/System.Transactions.Local/src/System/Transactions/Transaction.cs(503,0): at System.Transactions.Transaction.EnlistDurable(Guid resourceManagerIdentifier, IEnlistmentNotification enlistmentNotification, EnlistmentOptions enlistmentOptions)
        /_/src/libraries/System.Transactions.Local/tests/OleTxTests.cs(504,0): at System.Transactions.Tests.OleTxTests.OleTxFixture..ctor()
        /_/src/mono/System.Private.CoreLib/src/System/Reflection/ConstructorInvoker.Mono.cs(20,0): at System.Reflection.ConstructorInvoker.InterpretedInvoke(Object obj, Span`1 args, BindingFlags invokeAttr)
        ----- Inner Stack Trace #2 (Xunit.Sdk.TestClassException) -----
        
```

**Comment by lambdageek** [comment]
I don't think this is a supported scenario.  Please disable the test

**Comment by AaronRobinsonMSFT** [comment]
Fixed with #74433
